src/configobj/__init__.py

```diff
@@ -757,7 +758,7 @@ def merge(self, indict, decoupled=False):
         """
         for key, val in list(indict.items()):
             if decoupled:
-                val = copy.deepcopy(val)
+                val = copy(val)
```

Comment:
As a side note, the unit tests passed after I changed `deepcopy` to `copy.copy`. If that change is breaking, then there should be a unit test to reflect that. No blame, just trying to avoid anyone else falling into the same trap.